app/controllers/solidus_klarna_payments/api/base_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module SolidusKlarnaPayments
+  module Api
+    class BaseController < ::Spree::Api::BaseController
```

Comment:
I don't get why you have restored this class.
It looks to me that `verify_authenticy_token` it's only required for the not-API controller.

Reply:
but both controllers are API now, no? so we need to skip that step